### PR TITLE
FP21-84:I replace text “Loading…” with animation

### PR DESCRIPTION
**Pull request by hassou2n**
# Description

I replace text “Loading…” with animation, if the tools are loaded, the user will see the animation.

Fixes # (issue)

# How to test?

Run the client  and server side.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/ToolsGrid/ToolsGrid.component.js [hunk 1]
```diff
@@ -32,19 +32,19 @@ export const ToolsGrid = ({ tools, isLoading }) => {
...
       })}
-    </>
+    </div>
   );
 
   return (
     <div>
       <Sorting setSelected={setSelected} />
-      <div className="grid-tools-container">{toolsToRender}</div>
+      <div>{toolsToRender}</div>
```
Comment (miltonhultgren) [inline packages/client/src/components/ToolsGrid/ToolsGrid.component.js:50]:
```suggestion
      {toolsToRender}
```

File: packages/client/src/components/Loader/Loader.component.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
```
Comment (miltonhultgren) [inline packages/client/src/components/Loader/Loader.component.js:19]: Not to be harsh, but you should never have code like this. 
Instead you probably want to set a size on the `lds-spinner` class instead to get the minimum size you want.

**Review by miltonhultgren — APPROVED** [review]
I didn't check the CSS or run this in my local machine.

**Review by TatjanaOcedova — APPROVED** [review]

File: packages/client/src/components/Loader/Loader.component.js (hunk 2 above)
Comment (hassou2n) [reply packages/client/src/components/Loader/Loader.component.js:19]: Really thank you for your advice, I will do it without using the `div` frequently.

**Pull request merged**